Sources/MapboxCoreNavigation/NavigationService.swift

```diff
@@ -148,12 +152,12 @@ public class MapboxNavigationService: NSObject, NavigationService {
     /**
      The active router. By default, a `RouteController`.
      */
-    public var router: Router!
+    public var router: Router { _router! }
     
     /**
      The events manager. Sends telemetry back to the Mapbox platform.
      */
-    public var eventsManager: NavigationEventsManager!
+    public var eventsManager: NavigationEventsManager { _eventsManager! }
```

Comment:
This and other changes to the public API in this pull request are technically backwards-incompatible, since it’s possible for application code to perform a `nil` check that would compile previously but not anymore. A short blurb in the changelog about these properties would be nice.

Reply:
Done